dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1339,20 +1339,32 @@ class CompileTest {
     return inDirectory<TaskResult>(testDirectory, () async {
       await flutter('packages', options: <String>['get']);
 
-      final Map<String, dynamic> compileRelease = await _compileApp(reportPackageContentSizes: reportPackageContentSizes);
-      final Map<String, dynamic> compileDebug = await _compileDebug(
+      // "initial" compile required downloading and creating the `android/.gradle` directory while "full"
+      // compiles only run `flutter clean` between runs.gi
+      final Map<String, dynamic> compileInitialRelease = await _compileApp(deleteGradleCache: true);
```

Comment:
Is there any way that the initial run can be taken only as far as needed to populate the cache?

Reply:
I believe that only the build command triggers the gradle steps that populates the .gradle directory. We could make a flag or command to just populate things like this, but it would likely not be of much use outside of our tests.